### PR TITLE
Fix array values in the "EmployeeRefPresenter"

### PR DESCRIPTION
**Pull request by ghost**
# Contribution checklist
Before:
files

After:
Screenshot 2023-03-30 at 11 34 13
Screenshot 2023-03-30 at 11 34 23

## Checklist

* [ ] - UI test added to added/changed functionality?
* [x] - Screenshot is added to PR if applicable ?
* [x] - Does the code work? Check whether function and logic are correct.
* [ ] - Does Changelog.md is updated with changes?
* [x] - Does the translations are up to date?
* [x] - Does it well tested?
* [x] - Tested for Chrome.
* [x] - Tested for Safari.
* [x] - Go through the changed code looking for typos, TODOs, commented LOCs, debugging pieces of code, etc.
* [x] - Rebase your branch onto master and upstream branch
* [ ] - Is there any redundant or duplicate code?
* [ ] - Are required links are linked to PR?
* [ ] - Does new code is well documented ?

## Related issues

A list of closed updated issues

### CODE REVIEW

File: plugins/contact-resources/src/components/EmployeeRefPresenter.svelte [hunk 1]
```diff
@@ -3,49 +3,31 @@
...
-  import { employeeByIdStore } from '../utils'
-  import AssigneeBox from './AssigneeBox.svelte'
-  import EmployeePresenter from './EmployeePresenter.svelte'
+  import EmployeeAttributePresenter from './EmployeeAttributePresenter.svelte'
 
-  export let value: Ref<Employee> | null | undefined
+  export let value: Ref<Employee> | Ref<Employee>[] | null | undefined
   export let kind: ButtonKind = 'link'
   export let tooltipLabels: PersonLabelTooltip | undefined = undefined
   export let onChange: ((value: Ref<Employee>) => void) | undefined = undefined
```
Comment (haiodo) [inline plugins/contact-resources/src/components/EmployeeRefPresenter.svelte:11]: @BykhovDenis Why we have onChange in Presenter? It seems I've missed when it was added.

Comment (BykhovDenis) [reply plugins/contact-resources/src/components/EmployeeRefPresenter.svelte:11]: @haiodo It allow us to change value in table or list

Comment (haiodo) [reply plugins/contact-resources/src/components/EmployeeRefPresenter.svelte:11]: @BykhovDenis it should not be presenter, it should be editor.

**Review by haiodo — APPROVED** [review]

**Comment by haiodo** [comment]
@sogorelkov Please sign all commits or recreate.

**Pull request merged**